.github/workflows/publish.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+changed_version="$(git diff --name-only HEAD~1 HEAD -- VERSION)"
+
+if [[ "${changed_version}" ]]; then
+  echo "Publishing because the version changed since the last commit"
+
+  git config user.name "${GITHUB_ACTOR}"
+  git config user.email "${GITHUB_ACTOR}@users.noreply.github.com"
+  git tag "v$(shell cat VERSION)"
```

Comment:
Does this strip the potential newline in the `VERSION` file?

Reply:
It looks like `$(...)` strips it.